pkg/util/mount/mount_linux.go

```diff
@@ -41,19 +41,19 @@ const (
 	procMountsPath = "/proc/mounts"
 )
 
-// Mounter provides the default implementation of mount.Interface
+// LinuxMounter provides the default implementation of mount.Mounter
 // for the linux platform.  This implementation assumes that the
 // kubelet is running in the host's root mount namespace.
-type Mounter struct{}
+type LinuxMounter struct{}
```

Comment:
Since this symbol would now be different on a different OS, it's not very useful as an exported symbol